add how to update Latigo on cluster to README (#70)
equinor/latigo: README.md

make req
```

### How to update Latigo on the k8s cluster

Note: this repo has CI as the GitHub actions that are in the `.github/workflows/` folder.

#### Make new Latigo version
If you want to push your changes to k8s:
- increase version in `VERSION` file along with your PR;
- merge PR to `master`;

> - wait until GitHub actions will build and push the new image.

Reviewer: ```suggestion
- wait until GitHub actions build and push the new image.
```

#### Change version of Latigo in k8s configs

> After new image will be pushed to the docker registry replace old version with the new one in this [repo](https://github.com/equinor/latigo-k8s) in following files:

Reviewer: ```suggestion
After new image is pushed to the docker registry replace the old version with the new one in this [repo](https://github.com/equinor/latigo-k8s) in following files:
```

Reviewer: Also, a note: `images` section of `kustomization.yaml` allows to define the images in just one place, like this: https://github.com/equinor/omnia-prevent-infrastructure/blob/master/kube/base/kustomization.yaml#L9
